vm/adb/adb.go

```diff
@@ -77,7 +83,7 @@ func ctor(env *vmimpl.Env) (vmimpl.Pool, error) {
 	}
 	devRe := regexp.MustCompile("[0-9A-F]+")
 	for _, dev := range cfg.Devices {
-		if !devRe.MatchString(dev) {
+		if !devRe.MatchString(dev.Serial) {
```

Comment:
In this PR a serial number can also be of an IP:PORT form, but the regular expression above seems to intend to only accept regular android device ids. If it accepts IPs as well, it is only because the regexp itself is invalid - it should have been `^[0-9A-F]+$`.

Please adjust the regexp so that it only accepts either valid regular serial numbers, or a valid cuttlefish id (which is IP:port, as I understand).